### PR TITLE
Bug 1618816 - BONUS: Define json schema for `log_details` column

### PR DESCRIPTION
**Pull request by octavian-negru**
[Back to Bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1618816)

### CODE REVIEW

File: schemas/log-details.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "type": "object",
```
Comment (ionutgoldan) [inline schemas/log-details.json]: We should also provide `title` and `description` to this schema, so it's clearer for other developers.

Comment (ionutgoldan) [reply schemas/log-details.json]: Oh, and the Travis tests require some fixing.

Comment (octavian-negru) [reply schemas/log-details.json]: Done. Not sure I added the best description.

**Comment by bebef1987** [comment]
@sarah-clements  can you review this please

**Pull request merged**